### ISSUE TITLE
Cookies parsing problem in httpx 0.14.0

### ISSUE THREAD

**Issue by andredias** [encode/httpx#1162]
After I updated httpx from version 0.13.3 to 0.14.0, I've got problems with cookie parsing.

On httpx 0.13.3, if I get this header (Response.headers):
```
(Pdb) resp.headers
Headers([('content-length', '80'), ('content-type', 'application/json'),
('set-cookie', 'session_id=7QXeu-R7ERoyL4H0XSkHRg; HttpOnly; Path=/; SameSite=lax; Secure'),
('set-cookie', 'csrf="ezkpjJeJTfgNjmOt2Mgh/nk4/3IsxGZN4zglr0IQbfI="; Path=/; SameSite=lax; Secure')])
(Pdb) resp.cookies.jar
<CookieJar[Cookie(version=0, name='csrf', value='"ezkpjJeJTfgNjmOt2Mgh/nk4/3IsxGZN4zglr0IQbfI="', port=None,
port_specified=False, domain='testserver.local', domain_specified=False, domain_initial_dot=False, path='/',
path_specified=True, secure=True, expires=None, discard=True, comment=None, comment_url=None,
rest={'SameSite': 'lax'}, rfc2109=False), Cookie(version=0, name='session_id', value='7QXeu-R7ERoyL4H0XSkHRg',
port=None, port_specified=False, domain='testserver.local', domain_specified=False, domain_initial_dot=False,
path='/', path_specified=True, secure=True, expires=None, discard=True, comment=None, comment_url=None,
rest={'HttpOnly': None, 'SameSite': 'lax'}, rfc2109=False)]>
(Pdb) resp.cookies['csrf']
'"ezkpjJeJTfgNjmOt2Mgh/nk4/3IsxGZN4zglr0IQbfI="'
``` 

These are the dependencies:

```
httpx 0.13.3 The next generation HTTP client.
├── certifi *
├── chardet >=3.0.0,<4.0.0
├── hstspreload *
├── httpcore >=0.9.0,<0.10.0
│   ├── h11 >=0.8,<0.10 
│   ├── h2 >=3.0.0,<4.0.0 
│   │   ├── hpack >=3.0,<4 
│   │   └── hyperframe >=5.2.0,<6 
│   └── sniffio >=1.0.0,<2.0.0 
├── idna >=2.0.0,<3.0.0
├── rfc3986 >=1.3,<2
└── sniffio *
```

----

However, after I updated to 
```
httpx 0.14.0 The next generation HTTP client.
├── certifi *
├── chardet >=3.0.0,<4.0.0
├── httpcore >=0.10.0,<0.11.0
│   ├── h11 >=0.8,<0.10 
│   └── sniffio >=1.0.0,<2.0.0 
├── idna >=2.0.0,<3.0.0
├── rfc3986 >=1.3,<2
└── sniffio *
```

This is what I get now from a Response.headers and a Response.cookies.jar:
```
(Pdb) resp.headers
Headers([('content-length', '80'), ('content-type', 'application/json'),
('set-cookie', 'session_id=gSZ-G9WyYFLx6EYaqEtChw; HttpOnly; Path=/; SameSite=lax; Secure'),
('set-cookie', 'csrf="9NflHETs8dPrkxX1sopAHklYIkQTgrVqUHP9gu4pRuo="; Path=/; SameSite=lax; Secure')])
(Pdb) resp.cookies.jar
<CookieJar[Cookie(version=0, name='session_id', value='gSZ-G9WyYFLx6EYaqEtChw', port=None,
port_specified=False, domain='testserver.local', domain_specified=False, domain_initial_dot=False, path='/',
path_specified=True, secure=True, expires=None, discard=True, comment=None, comment_url=None,
rest={'HttpOnly': None, 'SameSite': 'lax', 'Secure, csrf': '"9NflHETs8dPrkxX1sopAHklYIkQTgrVqUHP9gu4pRuo="'},
rfc2109=False)]>
(Pdb) resp.cookies['csrf']
*** KeyError: 'csrf'
```

There is a key error even though `csrf` is present in the header.

**Comment by j178** [comment]
Probably the same problem as #1151, you could try install from the latest github repo and see if the problem resolved:
```sh
$ pip install https://github.com/encode/httpx/archive/master.zip
```

**Comment by andredias** [comment]
Duplicated of #1151

**Comment by tomchristie** [comment]
Now resolved in 0.14.1.

Thanks for the report!
